source/connection-monitoring-and-pooling/tests/pool-close.yml

```diff
@@ -0,0 +1,10 @@
+version: 1
+style: unit
+description: A Pool MUST be able to be manually closed.
+operations:
+  - name: close
+events:
+  - type: ConnectionPoolCreated
+    address: "42"
```

Comment:
I think it would be easier for most languages if we stuck to `42` as an integer rather than also using as a string (like here)